packages/interface/src/cw1_subkeys.rs

```diff
@@ -30,3 +32,89 @@ impl<Chain: CwEnv> Uploadable for Cw1SubKeys<Chain> {
         )
     }
 }
+
+#[cfg(not(target_arch = "wasm32"))]
+/// Copy messages of the contract to implement cw-orch helpers on Execute([`cw_orch::ExecuteFns`]) and Query([`cw_orch::QueryFns`]) interfaces
+mod interfaces {
+    use super::*;
+
+    use cosmwasm_schema::schemars::JsonSchema;
+
+    #[derive(cw_orch::ExecuteFns, from_interface_derive::FromInterface)]
+    enum ExecuteMsgInterface<T = Empty>
+    where
+        T: Clone + std::fmt::Debug + PartialEq + JsonSchema,
+    {
+        /// Execute requests the contract to re-dispatch all these messages with the
+        /// contract's address as sender. Every implementation has it's own logic to
+        /// determine in
+        // This method is renamed to not conflict with [`CwOrchExecute::execute`]
+        #[renamed(Execute)]
```

Comment:
For those, can we use the `cw_orch::rename` attribute directly instead of this new attribute ?
This allows for keeping the usual flow and is more understandable for people reading this code 
Something like the following : 
```rust
#[cw_orch(fn_name(execute_requests))]
Execute{
            msgs: Vec<cosmwasm_std::CosmosMsg<T>>,
},
```

Reply:
Oh, didn't know `fn_name` exist! It simplifies this thing a lot, thank you!